qa/pull-tester/rpc-tests.py

```diff
@@ -247,7 +247,8 @@ def option_passed(option_without_dashes):
     'getlogcategories',
     'getrawtransaction',
     Disabled('electrum_basics', "Needs to be skipped if electrs is not built"),
-    Disabled('electrum_reorg', "Needs to be skipped if electrs is not built")
+    Disabled('electrum_reorg', "Needs to be skipped if electrs is not built"),
+    Disabled('electrum_shutdownonerror', "Needs to be skipped if electrs is not built")
```

Comment:
Wonder if we could detect if `electrs` has been built and run the above tests only in this case. Something like we do to detect if wallet support has been built, i.e.  `ENABLE_WALLET`

Reply:
Fixed in https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/1869